src/changes-backwards-incompatible.apib

```diff
@@ -3,6 +3,10 @@
 
 We list all backwards-incompatible changes here. As described above, new additions and forwards-compatible changes don’t need a new API version and can be found [here](#additions).
 
+#### Latest
```

Comment:
B : we should pick a new version number...

Reply:
What's the logic for upcoming version numbers? Which date do we pick?

Reply:
Is it always the wednesday in the guild week?